### DISCUSSION TITLE
How to use nginx to reverse proxy nuxt3 with 5 different paths?

### DISCUSSION THREAD

**Discussion by bbhxwl** [nuxt/nuxt#27739] [Questions]
How to use nginx to reverse proxy nuxt3 with 5 different paths?
The following configuration cannot be accessed properly. Because the resource files also access the root directory.

```
location /web1/ {
        proxy_pass http://location:5001/;
        proxy_next_upstream error timeout invalid_header http_500 http_502 http_503 http_504;
        proxy_next_upstream_tries 2;
    }
location /web2/ {
        proxy_pass http://location:5002/;
        proxy_next_upstream error timeout invalid_header http_500 http_502 http_503 http_504;
        proxy_next_upstream_tries 2;
    }
location /web3/ {
        proxy_pass http://location:5003/;
        proxy_next_upstream error timeout invalid_header http_500 http_502 http_503 http_504;
        proxy_next_upstream_tries 2;
    }
```

**Comment by sukhcha1n** [comment]
Have you tried changing the base url in vite config?

**Comment by warflash** [comment]
As @sukhcha1n mentioned, nuxt's [app.baseUrl](https://nuxt.com/docs/api/nuxt-config#baseurl) should be what you're looking for

**Comment by facularrauri** [comment]
Any updates? I want to change nuxt's [app.baseUrl](https://nuxt.com/docs/api/nuxt-config#baseurl) at runtime depending on the host

**Reply by Saeid-Za** [reply]
You could also use an environment variable to set the `baseUrl`:
```bash
NUXT_APP_BASE_URL=/prefix/ node .output/server/index.mjs
```
